roles/configure_apache/defaults/main.yml

```diff
@@ -1,7 +1,15 @@
 ---
 apache:
-    docroot: '{{apache_app_path |default("/var/www/html")}}'
-    servername: "{{ ansible_hostname }}"
+    docroot: '{{ apache_app_path |default("/var/www/html")}}'
+    servername: '{{ apache_servername | default(ansible_hostname) }}'
     directory_options: ''
-apache_listen_port: 80
-apache_app_url: '/'
\ No newline at end of file
+    listen_port: '{{ apache_port | default("80") }}'
```

Comment:
nice, I like the use of `default()` here so that defaults are actually kept in this file, instead of in the template file.

Reply:
Yeah, seems much better. Do you think it could just be `port` instead of `listen_port` ? Is `port` ambiguous in this context?

Reply:
i can't think of any other meaning in this context, so `port` seems fine to me?